trieste/acquisition/function/function.py

```diff
@@ -468,6 +468,84 @@ def acquisition(x: TensorType) -> TensorType:
     return acquisition
 
 
+class FastConstraintsFeasibility(SingleModelAcquisitionBuilder[ProbabilisticModel]):
+    """
+    Uses the :func:`fast_constraints_feasibility` function to build a
```

Comment:
I don't think this needs to mention that it uses `fast_constraints_feasibility` (which after all is an implementation detail)?

```suggestion
    Builder for a probability of feasiblity acquisition function that uses the residuals of 
    direct constraints defined in the search space.
```

Reply:
I copied that style from ProbabilityOfFeasibility, but I can remove it.

Reply:
Done in 1e41a57.